### ISSUE TITLE
Raw citations

### ISSUE THREAD

**Issue by DOFfactory** [jgm/pandoc#2427]
Is there a way how to let pandoc be the LaTeX citations as they are? Pandoc automatically converts LaTeX cite command into markdown:

> `\cite{aaa_2015}`

into

> `[@aaa_2015]`

Is it possible to let the whole `\cite{aaa_2015}` be as it is in the markdown too? Is there a way, which I cannot find in the manual?

**Comment by jgm** [comment]
If your source is Markdown, use

    pandoc -f markdown-citations

to turn off the citation recognition, and it will be parsed
as raw LaTeX.

+++ DOFfactory [Oct 01 15 03:20 ]:

**Comment by DOFfactory** [comment]
Thanks for the reply. Well, my source is (rather understandably) LaTeX. But I tried to convert to pandoc with the command 

> pandoc -f latex -t markdown-citations

It produces a correct markdown file, preserving all the `\cite{aaa_2015}` commands. I thought these are all treated now in pandoc as normal text. I tried to convert it further (docx, odt, etc.), and all these references are gone. Is there a way how to preserve them in these files? I tried to play with the raw_tex extension, but no luck.
